universal-login-wallet/cypress/support/index.js

```diff
@@ -0,0 +1,2 @@
+import './commands';
+import './LoginApp/commands';
```

Comment:
Why don't we name these commands just `loginPageCommands` and move it to `universal-login-wallet/cypress/support/loginPageCommands.js`? Then this file would be like this:
```js
import './commands';
import './loginPageCommands';
```